science/uhd/Portfile

```
@@ -22,15 +21,13 @@ if {${subport} eq ${name}} {
     long_description ${description}: \
 Provides the release version, which is typically updated every month or so.
 
-    github.setup EttusResearch uhd 3.15.0.0 v
-    # Change github.tarball_from to 'releases' or 'archive' next update
-    github.tarball_from tarball
-    checksums rmd160 ac9ef6473cd5b034650e3979236d2535f398dec2 \
-              sha256 fe385e826b8fbf4e869b4b919b7b7b8f01fba283bc0b8ffb954947f819b9c07b \
-              size   12896898
+    github.setup EttusResearch uhd 4.8.0.0 v
+    checksums rmd160 8efc172de52be82dcfeb64a306fad14be93cb361 \
+              sha256  a2159491949477dca67f5a9b05f5a80d8c2b32e91b95dd7fac8ddd3893e36d09 \
+              size    41358938
     revision  7
```

When updating the `version`, the `revision` should drop to `0`.
```suggestion
    revision  0
```